### ISSUE TITLE
Wrapping a third party library using an Unreal Engine Plugin

### ISSUE THREAD

**Issue by JohnJFarrow** [JohnJFarrow/UnrealcodeIssues#17]
Please leave any feedback about [Wrapping a third party library using an Unreal Engine Plugin](https://www.unrealcode.net/WrappingALibrary.html)

**Comment by booomji** [comment]
What a fantastic beginners guide to wrapping a third party library.
Thank you sir !

I hope you can do an example with the opencv library.  OpenCV & UE tutorials are old and outdated and would be welcome addition to how to get started using UE 5.3 onwards.
Cheers

**Comment by whybazinga** [comment]
Thank you for the article!
I haven't seen anyone else mention that `RuntimeDependencies.Add` does NOT copy anything (and that the comment above it is misleading) in their guides, had to find it out on my own, so it was very nice seeing that you've explained this issue and provided a solution.

**Comment by ArcherSmythe** [comment]
Thank you! This fills some of the gaps left in Epic's own documentation.  One question: If I'm using a library compiled externally, should I delete everything in Plugins/GeneratedExample/Source/GeneratedExample?

**Comment by JohnJFarrow** [comment]
No, you still need stuff like GeneratedExample.h which is the unreal module to do things like delayed loading
